jina/peapods/runtimes/zmq/zed.py

```diff
@@ -138,7 +138,7 @@ def _pre_hook(self, msg: 'Message') -> 'ZEDRuntime':
 
         info_msg = f'recv {msg.envelope.request_type} '
         if self.request_type == 'DataRequest':
-            info_msg += f'({self.envelope.header.exec_endpoint}) '
+            info_msg += f'({self.envelope.header.exec_endpoint}) - ({self.envelope.request_id}) '
```

Comment:
Building this `info_msg` seems rather expensive here? It looks ugly but it may be worth to check if `self.logger` would actually log the message and only construct it then? I think doing string operations on every data request for debugging purposes is not the best idea?

Reply:
that makes sense @jacobowitz, may u do a PR to fix this?